### PR TITLE
Add quickstart

### LINKED ISSUE #236: Create a new quick start section
The elemental docs are missing a quick start guide for new comers. We should write a simple quick start guide inspired by the confluence [How to Teal](https://confluence.suse.com/display/CECO/Elemental+Teal+HowTo) docs.

### PR DESCRIPTION
**Pull request by Itxaka**
Fixes #236 

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+## Introduction
+
+---
+
+#### What is Elemental Teal ?
+
+Elemental Teal is the combination of "SLE Micro for Rancher" with the Rancher Elemental stack
+
+SLE Micro for Rancher is a containerized and "stripped to the bones" operating system layer. It only contains grub2, dracut, a kernel, and systemd.
```
Comment (kkaempf) [inline docs/quickstart.md:13]:
```suggestion
SLE Micro for Rancher is a containerized and "stripped to the bones" operating system layer. It only requires grub2, dracut, a kernel, and systemd.
```

**Review by kkaempf — APPROVED** [review]
🤩

**Comment by Itxaka** [comment]
draft for now I want to add some more niceties to the page

**Comment by Itxaka** [comment]
Would recommend downloading the path and building the docs locally to see them in their full glory. Tabs and alternative methods are awesome.

Also disabled the stupidly sad grey theme and allowed to toggle between light and dark themes. What the hell is with that grey theme, so dull ¬_¬

**Pull request merged**